administrator/language/de-DE/mod_popular.ini

```diff
@@ -8,12 +8,23 @@ MOD_POPULAR="Beiträge – Beliebte"
 MOD_POPULAR_CREATED="Erstellt"
 MOD_POPULAR_FIELD_AUTHORS_LABEL="Autoren"
 MOD_POPULAR_FIELD_COUNT_LABEL="Anzahl"
+MOD_POPULAR_FIELD_DATEFILTERING_LABEL="Datumsfilter"
+MOD_POPULAR_FIELD_DATERANGEFIELD_LABEL="Datumsbereich"
+MOD_POPULAR_FIELD_ENDDATE_LABEL="bis Datum"
+MOD_POPULAR_FIELD_RELATIVEDATE_LABEL="Relatives Datum"
+MOD_POPULAR_FIELD_STARTDATE_LABEL="Datumsbereich von"
 MOD_POPULAR_FIELD_VALUE_ADDED_OR_MODIFIED_BY_ME="Von mir erstellt oder bearbeitet"
 MOD_POPULAR_FIELD_VALUE_ANYONE="Jeder"
 MOD_POPULAR_FIELD_VALUE_NOT_ADDED_OR_MODIFIED_BY_ME="Nicht von mir erstellt oder bearbeitet"
 MOD_POPULAR_ITEMS="Beliebteste Beiträge"
 ; The following string is deprecated and will be removed with 7.0
 MOD_POPULAR_NO_MATCHING_RESULTS="Keine übereinstimmenden Ergebnisse"
+MOD_POPULAR_OPTION_CREATED_VALUE="Erstellungsdatum"
+MOD_POPULAR_OPTION_DATERANGE_VALUE="Datumsbereich"
```

Comment:
```suggestion
MOD_POPULAR_OPTION_DATERANGE_VALUE="Zeitraum"
```

Reply:
Wir verwenden bisher Datumsbereich.
Allerdings ist die Änderung sinnvoll. Sollte aber in einem Clean-Up-PR gemacht werden, für alle Vorkommnisse.